packages/types/src/codec/Enum.ts

```diff
@@ -2,129 +2,6 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { isString, isU8a, u8aToHex, assert } from '@polkadot/util';
+import Enum from './EnumType';
```

Comment:
For now. Should rename and not create chaos like the last time around - as it stands, keeping it like this until all-ok here.